app/src/main/java/de/tum/in/tumcampusapp/utils/DateTimeUtils.kt

```diff
@@ -42,14 +42,15 @@ object DateTimeUtils {
     }
 
     /**
-     * Format a past timestamp with degrading granularity
+     * Format a *past* ISO string timestamp with degrading granularity
+     * This is similar to printFutureTime(), but not specialized on future dates
      */
-    fun getTimeOrDayISO(datetime: String, context: Context): String {
+    fun printTimeOrDayFromISO(datetime: String, context: Context): String {
         val d = DateTimeUtils.parseIsoDate(datetime) ?: return ""
-        return DateTimeUtils.getTimeOrDay(d, context)
+        return DateTimeUtils.printTimeOrDay(d, context)
     }
 
-    fun getTimeOrDay(time: DateTime, context: Context): String {
+    fun printTimeOrDay(time: DateTime, context: Context): String {
```

Comment:
As with future time, the name of this method should better reflect what it does. Additionally, please add example results in the documentation.